libs/sdk-backend-spi/src/user/index.ts

```diff
@@ -1,6 +1,39 @@
 // (C) 2020 GoodData Corporation
+import { ObjRef } from "@gooddata/sdk-model";
 import { IUserSettingsService } from "./settings";
 
+/**
+ * Represents platform user
+ *
+ * @alpha
+ */
+export interface IUser {
+    /**
+     * Stored user reference
+     */
+    ref: ObjRef;
+
+    /**
+     * Login
```

Comment:
We should probably say what this is. Is it a unique user ID or what should it be?